<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Ostinato</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="ostinato.qrc" >:/icons/about.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="separator" />
    <addaction name="actionPreferences" />
    <addaction name="actionFileExit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout" />
    <addaction name="actionAboutQt" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionFileExit" >
   <property name="icon" >
    <iconset resource="ostinato.qrc" >:/icons/exit.png</iconset>
   </property>
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionHelpAbout" >
   <property name="icon" >
    <iconset resource="ostinato.qrc" >:/icons/about.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionPreferences" >
   <property name="icon" >
    <iconset resource="ostinato.qrc" >:/icons/preferences.png</iconset>
   </property>
   <property name="text" >
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="icon" >
    <iconset resource="ostinato.qrc" >:/icons/qt.png</iconset>
   </property>
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ostinato.qrc" />
 </resources>
 <connections/>
</ui>
